### DISCUSSION TITLE
MacOS 12 has been shipped according to the Github Public Roadmap but not working yet

### DISCUSSION THREAD

**Discussion by sukhrobkhakimov** [community/community#13552] [Actions] [answered]
For me, it shows `Waiting for a runner to pick up this job...` for hours
https://github.com/github/roadmap/issues/240#issue-991355895

**Comment by ethomson** [comment, marked as answer]
Hi @sukhrobkhakimov - it has shipped only in private beta. Is it possible that you are not in the beta program? If not, I'm afraid that you will need to wait for general availability of macOS 12.

**Reply by sukhrobkhakimov** [reply]
@ethomson Thank you for the reply. I appreciate it.
- How to participate in the beta program? I would be happy to be part of it.
- When will it be released to the public?

**Reply by Rotemsim3339** [reply]
Hi @ethomson,
I have the same issue with the runner.  I would be happy to be part of the beta program as well, how can we join?
And we really need to know when macOS 12 will be released to the public. we cant run xcode 13.3 because of it.
Thanks!

**Reply by jaredh159** [reply]
just tested an action workflow this morning, and i think this is out of private beta now, because a runner picked up a job labeled `macos-12` for me right away. 👍
